model/source.go

```diff
@@ -41,6 +43,34 @@ type Source struct {
 	Endpoints        []Endpoint
 }
 
+func (src *Source) AsJSON() []byte {
```

Comment:
This isn't getting called - can it be removed?

Reply:
sure, it was leftover - good eye.